### DISCUSSION TITLE
0.26.0 release date

### DISCUSSION THREAD

**Discussion by Homomorpheus** [pyodide/pyodide#4751] [Q&A] [answered]
ryanking13 stated in https://github.com/pyodide/pyodide/discussions/4737 that 0.26.0 will be released "in a few weeks". May I ask whether or not that will be before July? If not, will there be another pre-release until July?

**Comment by hoodmane** [comment, marked as answer]
We are hoping to do it in the next 7 days, I'd prefer by Monday.

**Reply by Homomorpheus** [reply]
Thank you for your answer!
